share/spice/git_book_status/git_book_status.css

```diff
@@ -0,0 +1,25 @@
+.zci--git_book_status .gbstatus--header {
+    font-size: 1.5em;
```

Comment:
We have a `t-xl` class (see https://duckduckgo.com/styleguide#txt-n-color) that can be used for the font-size. I think should also be a class for `font-size: 1em` as well